### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by dandelionadia**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 import Index from './pages/artist/Index'
 import Home from './pages/home/Home'
 import Album from './pages/album/Album'
+import NavBar from './molecules/NavBar'
+import MenuMobile from './atoms/MenuMobile'
+
+const areaMobile = `
+  menuMobile
+  content
+  / auto
```
Comment (kettanaito) [inline src/App.js]: If you remove "auto", how wide will the component be? Would that be acceptable?

File: src/atoms/Menu.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { Composition } from 'atomic-layout'
+import styled from 'styled-components'
+import { withRouter } from 'react-router-dom'
+import Link from './Link'
+
+const StyledUl = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  // display: block !important;
```
Comment (kettanaito) [inline src/atoms/Menu.js]: Please remove the commented code, if you don't use it.

File: src/atoms/MenuMobile.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  left: 0;
+  top: 0;
+  display: flex;
+  align-items: flex-start;
+`
+
+const StyledMenu = styled.div`
+  margin-left: ${({ isOpen }) => (isOpen ? '0' : '-200px')};
+  /* visibility: ${({ isOpen }) => (isOpen ? 'visible' : 'hidden')}; */
+  /** @todo Finish tomorrow */
```
Comment (kettanaito) [inline src/atoms/MenuMobile.js]: Please remove the commented code if you don't use it.

File: src/atoms/MenuMobile.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  height: 3px;
+  margin: 0 0 5px 0;
+`
+
+const MenuBtn = ({ onClick }) => {
+  return (
+    <StyledMenuBtn onClick={onClick}>
+      <StyledMenuBtnLine></StyledMenuBtnLine>
+      <StyledMenuBtnLine></StyledMenuBtnLine>
+      <StyledMenuBtnLine></StyledMenuBtnLine>
```
Comment (kettanaito) [inline src/atoms/MenuMobile.js]: You can make this component self-closing:

```jsx
<Component />
```

File: src/App.js (hunk 1 above)
Comment (dandelionadia) [reply src/App.js]: Yes. It is okay.

File: src/atoms/Link.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import { NavLink } from 'react-router-dom'
+import Text from './Text'
+
+const areasMobile = `
+	icon name
+	/ auto 1fr
+`
+
+const StyledLink = styled(NavLink)`
+  // display: flex;
```
Comment (kettanaito) [inline src/atoms/Link.js]: Please remove commented code.

File: src/molecules/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    url: '/'
+  },
+  {
+    icon: GoSearch,
+    name: 'Search',
+    url: ''
+  }
+]
+
+const dataUser = [
```
Comment (kettanaito) [inline src/molecules/NavBar.js]: It's great that you separate a user data like this. Nice!

**Pull request merged**